lava/magma/core/model/c/ports_embedded.c

```diff
@@ -0,0 +1,16 @@
+// implementation of ports API on the embedded platform
+#define EMBEDDED
+#include "ports.h"
+
+int port_send(Port *port,void* data){
```

Comment:
Why not just send, recv, peek, probe for short like in Python API?
There are likely no other functions with the namespace when the user writes this code and the typed Port argument makes it pretty clear what it does.

Reply:
changed in https://github.com/lava-nc/lava/pull/31/commits/1caad0cbc61ce87d61ddb10a03b4ae18b506edc6